tests/unit/data/html/data.json

```diff
@@ -37,7 +37,10 @@
         "incorrect": "No <i>2</i>",
         "correct": "Yes <i>2</i>"
       },
-      "zone": "Zone <b>2</b>",
+      "zones": [
+        "Zone <b>2</b>",
+        "Zone <i>1</i>"
+      ],
```

Comment:
@haikuginger We should probably switch to using `"zones"` instead of `"zone"` for the other items as well. The same comment applies to `tests/unit/data/plain/data.json`.